test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/RazorPagesWithoutModel_DesignTime.ir.txt

```diff
@@ -49,15 +50,16 @@ Document -
                     IntermediateToken - (449:21,43 [1] RazorPagesWithoutModel.cshtml) - Html - >
                     IntermediateToken - (450:21,44 [6] RazorPagesWithoutModel.cshtml) - Html - \n    
                 TagHelper - (456:22,4 [31] RazorPagesWithoutModel.cshtml) - div - TagMode.StartTagAndEndTag
-                    TagHelperBody - 
-                    CreateTagHelper -  - DivTagHelper
-                    AddTagHelperHtmlAttribute -  - class - AttributeStructure.DoubleQuotes
+                    DefaultTagHelperBody - 
+                    DefaultTagHelperCreate -  - DivTagHelper
+                    DefaultTagHelperHtmlAttribute -  - class - HtmlAttributeValueStyle.DoubleQuotes
```

Comment:
It's just cosmetic. I made the IR serialization consistent. Some nodes were using one, some the other, but they mean the same thing